content/en/llm_observability/setup/auto_instrumentation.md

```diff
@@ -28,6 +28,7 @@ Datadog's [LLM Observability Python SDK][16] provides integrations that automati
 | [Amazon Bedrock](#amazon-bedrock)          | >= 1.31.57         | >= 2.9.0          |
 | [Anthropic](#anthropic)                    | >= 0.28.0          | >= 2.10.0         |
 | [Google Gemini](#google-gemini)            | >= 0.7.2           | >= 2.14.0         |
+| [Vertex AI](#vertex-ai)                    | >= 1.71.1          | >= 2.18.0         |
```

Comment:
Need to verify the tracer release version.

Reply:
Were you able to confirm this version?

Reply:
Yes, this is the correct version.